### DISCUSSION TITLE
How can I run this on mac os

### DISCUSSION THREAD

**Discussion by SolidRhino** [searx/searx#2756] [Installation issue: Docker]
I like to run my one installation on mac os. But when I run it and try to open the searchenging on "https://localhost/" I don't get a website. What I have done wrong?

**Comment by return42** [comment]
We do not have experience with macOS. If you you use the search function of github in this reposetory you will find some hints about how to install on MacOS.   As alternative it might be less complex using a docker image .. but MacOS is special and I can't help in deep.

**Reply by unixfox** [reply]
This issue was migrated from searx-docker (look at the category) so it was about how to run searx-docker on MacOS.

**Reply by return42** [reply]
Aargh, sorry / I'm an idiot.
